src/coreclr/pal/tools/gen-dactable-rva.sh

```diff
@@ -15,10 +15,12 @@ while read -r line; do
     # * longer than 16, capture last 16 characters.
     #
     array=($line)
-    value="$(printf "%016s\n" ${array[2]:(${#array[2]} > 16 ? -16 : 0)})"
+    value="$(printf "%s\n" ${array[2]:(${#array[2]} > 16 ? -16 : 0)})"
+    zeros="0000000000000000"
+    pvalue="$(printf "%s%s" "${zeros:${#value}}" "$value")"
```

Comment:
is this needed  because bash differences?

Reply:
Yes, no idea why, on some system (with the same bash version), there are spaces instead of zero's - hence I made this fix - maybe there's more clever way - or maybe those 0x0000.. are not needed, then simple `printf %s` can be used?

Reply:
This is just generating a C header file with a define and we only care about the value. This is the only usage:
https://github.com/dotnet/runtime/blob/d17741d0c44b13cb4df864e0184cf1e499edcd57/src/coreclr/debug/daccess/daccess.cpp#L6996

Reply:
Yes, I saw that, but I was thinking there's more magic behind that, as I'm no expert in this area, but simple `%s` is fine with me :)